### PR TITLE
Have a unit containing 'let x = t in' separated from the term in which x is bound

### PR DESCRIPTION
**Pull request by GuillaumeGen**
When trying to format the Nickel language using topiary, one issue we have is that the `let x =... in` and the term in which `x` is bound are at the same level in the CST tree, making it impossible (well, in the sense that it would require to develop a new feature that we do not have yet, not in the sense that there is no hope this feature is creatable) to keep the small variable binding single-lined.

This distinction would make it possible to have the code formatted as 
```
let x = 3 in
some_huge_term
```
rather than
```
let x =
  3
in
some_huge_term
```
where the multiple lines with very few characters hurt readability.

Hence this proposition to create a separate syntactical category for `let x = 3 in`.

### CODE REVIEW

File: grammar.js [hunk 1]
```diff
@@ -120,14 +120,18 @@ module.exports = grammar({
     ),
 
     let_expr: $ => seq(
+      field("var", $.let_in_block),
```
Comment (yannham) [inline grammar.js:123]: Nitpick: "var" is a bit off to talk about a binding `(var, term)` (plus other potential annotations).

**Review by yannham — APPROVED** [review]
Left a nitpick comment, but otherwise fine by me!

**Review by ErinvanderVeen — APPROVED** [review]

**Comment by yannham** [comment]
@GuillaumeGen feel free to do whatever you want with the comment, and ping me once the conversation is resolved, and I'll merge.

**Comment by GuillaumeGen** [comment]
Done @yannham

**Comment by yannham** [comment]
Argh, I merged too quickly...the CI was failing. I guess you have to substitute some `vars` for `binding` in the corpus. Doing that right now.

**Pull request merged**